### PR TITLE
Badges & button component

### PR DESCRIPTION
**Pull request by Rishabh-078**
Badges component contains different types of profile and icon badges
Button component contains different types of  solid, outlined, icon and float buttons

### CODE REVIEW

File: lib/components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+}
+
+.ava-round-img {
+    border-radius: 50%;
+}
+
+.badge-dot {
```
Comment (vanshitamoolchandani) [inline lib/components/Badge/badge.css:11]: You can make your badge look more distinct by adding white border to it.

**Comment by iamalhera** [comment]
looks good

File: documentation.html [hunk 2]
```diff
@@ -63,19 +63,19 @@ <h3>Avatar</h3>
                 <h5>Examples of Round Image Avatar:</h5>
                 <div class="container">
                     <div class="avatar">
-                        <img class="ava-round-img avatar-xxl" src="./assets/avatar.png" alt="Avatar" loading="lazy"/>
+                        <img class="ava-round-img avatar-xxl" src="./assets/avatar.png" alt="Avatar" loading="lazy" />
```
Comment (AnkitaBagale) [inline documentation.html]: use meaning class names 
instead of `ava-round-img` use `avatar-rounded`

File: documentation.html [hunk 3]
```diff
@@ -63,19 +63,19 @@ <h3>Avatar</h3>
                 <h5>Examples of Round Image Avatar:</h5>
                 <div class="container">
                     <div class="avatar">
```
Comment (AnkitaBagale) [inline documentation.html:65]: this div looks like a wrapper for avatar, use class name such as avatar-wrapper or avatar-container

File: documentation.html [hunk 4]
```diff
@@ -85,19 +85,19 @@ <h5>Examples of Round Image Avatar:</h5>
                 <h5>Examples of Square Image Avatar:</h5>
                 <div class="container">
                     <div class="avatar">
-                        <img class="ava-sq-img avatar-xxl" src="./assets/avatar.png" alt="Avatar" loading="lazy"/>
+                        <img class="ava-sq-img avatar-xxl" src="./assets/avatar.png" alt="Avatar" loading="lazy" />
```
Comment (AnkitaBagale) [inline documentation.html]: again use meaningful names - avatar-square

File: documentation.html [hunk 5]
```diff
@@ -154,6 +154,125 @@ <h5>Examples of Outline Alert Box:</h5>
...
+                    Along with it icon badges are also present.</p>
+                <h5>Examples of Dot Profile Badges:</h5>
+                <div class="container">
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-m" src="./assets/avatar.png" alt="Avatar" />
+                        <div class="badge-dot badge-online-a"></div>
+                    </div>
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-m" src="./assets/avatar.png" alt="Avatar" />
+                        <div class="badge-dot badge-offline-a"></div>
```
Comment (AnkitaBagale) [inline documentation.html]: badge-online-a, badge-online-b these names having a,b does not make much sense, please add more meaningful names instead.

File: index.css [hunk 6]
```diff
@@ -114,6 +114,10 @@ button {
     cursor: pointer;
 }
 
+.secondary-btn>a {
```
Comment (AnkitaBagale) [inline index.css]: try to avoid complex selectors to make debugging easier in large apps.
Try to apply CSS to single classes as much as possible

**Comment by AnkitaBagale** [comment]
Hey Rishabh,
It's great that you added PR description, please add screenshots as well next time if PR is about UI changes.
Added few comments, please check it.

**Review by AnkitaBagale — CHANGES_REQUESTED** [review]

File: lib/components/Badge/badge.css (hunk 1 above)
Comment (Rishabh-078) [reply lib/components/Badge/badge.css:11]: Thanks for the suggestion, has been implemented.

File: index.css (hunk 6 above)
Comment (Rishabh-078) [reply index.css]: Thank you for all the suggestions, have made the required changes.
Will work on naming classes more clearly.

**Comment by Rishabh-078** [comment]
Made changes according to the PR reviews.

**Pull request merged**